scipy/special/cephes/bdtr.c

```diff
@@ -148,103 +148,92 @@
 
 #include "mconf.h"
 
-double bdtrc(k, n, p)
-int k, n;
-double p;
+double bdtrc(double k, int n, double p)
```

Comment:
Looks like you should compute `floor(k)` somewhere at the beginning of the function and use it throughout.

Reply:
good idea, changed throughout to have a single `fk = floor(k)` call in each function w/subsequent calculations using `fk`.